.github/packager/js/index.js

```diff
@@ -0,0 +1,4 @@
+// Main entry point for the pactus-grpc package
+// Re-exports all modules from the pactus_grpc namespace
+
+module.exports = require('./pactus_grpc'); 
```

Comment:
No new lines at the end of the files. please install [EditorConfig](https://editorconfig.org/) and ensure we add new line at the end files.